### DISCUSSION TITLE
CBERS4A - WPM GRID.

### DISCUSSION THREAD

**Discussion by VictorPierobom** [gabriel-russo/cbers4asat#9] [General]
Hello everyone, 

I'm working with the CBERS4ASAT and I did a script to do the download of multiples WPM level 4 scenes. 

For write the first script I used the grid (and the path_row) of MUX sensor, that is similar, but not equal to the WPM grid. I need the shapefile of WPM GRID .  

 Someone have the WPM grid and can share with me? 

thanks.

**Comment by gabriel-russo** [comment]
From my search in INPE's resources, the WPM sensor uses the same grid of MUX sensor.

From that, all grids that i can found are here: www.dgi.inpe.br/documentacao/grades

If you're talking about the displacement of a raster on grid, i think it's normal... depends of the processing of geometric correction.

You can see more about that here: www.dgi.inpe.br/documentacao/arquivos/NiveisdeProcessamentoCBERSAMZ.pdf


_Note: I really recommend talk about this directly with INPE if it's really important_

**Reply by Pedro2290** [reply]
Olá, bom dia, Gabriel. Você pode me ajudar? esse link da documentação não é válido mais. Já tem um tempo que procura documentação sobre o cbers 4a (lembro que tive acesso uma vez a um material muito bom, mas perdi) mas não encontro em lugar nenhum. Você poderia me ajudar?
Obrigado!

**Reply by gabriel-russo** [reply]
Opa, beleza? realmente os links não estão funcionando, está bem complicado a situação da disponibilização dos serviços do inpe nesses últimos meses... realmente queria que alguem me explicasse por que tanta instabilidade... mas para resolver seu problema o mais breve possível eu bolei um "workaround" para baixar os grids direto do inpe, segue instruções:

1. Abra o Qgis e clique em "Nova conexão" no serviço WFS/OGC API - Features
2. Em URL insira este link: http://cdsr.dpi.inpe.br/geoserver/vector_data/wms?
3. Abra a conexão e clique na camada do grid que você quer
4. Clique com o botão direito na camada
5. Exportar > Guardar elementos como...

Com isso você vai poder baixar os grids neste momento, até o link [www.dgi.inpe.br/documentacao/grades](http://www.dgi.inpe.br/documentacao/grades) voltar pro ar novamente...

Ainda não testei mas provavelmente a função da biblioteca de baixar os grids deve não estar funcionando, então já fica avisado que o motivo é que os grids do site do dgi inpe estão indisponívels

**Reply by gabriel-russo** [reply]
>  [...] procura documentação sobre o cbers 4a

Sobre a documentação, ainda estou procurando, mas eu tenho esta aparesentação aqui que peguei emprestado sobre detalhes das imagens do cbers:

https://drive.google.com/file/d/1KM-s0N6CA2NlO-O2Tp5Vtnqzi-aA8BDP/view?usp=drive_link

**Comment by VictorPierobom** [comment]
Hello again, 

in the first moment I assumed that the both grid were equal, but reading the documentation is possible understand that the WPM sensor have a smaller grid. The MUX have 95km of width, while WPM have 92km. 

source: https://forest-gis.com/2019/12/novo-satelite-do-tipo-cbers-e-lancado-com-sucesso-na-china.html/

I will send a e-mail to INPE and ask about this information, when receive the feedback a post here the answer. 

Thanks.

**Reply by VictorPierobom** [reply]
a better source: http://www.cbers.inpe.br/sobre/cameras/cbers04a.php

**Comment by BerniniH** [comment]
Hi, 

For MUX, CBERS4ASAT delineate a search for image based on path/row that should be the same in WPM. The Grid difference between MUX/WPM has no effect to the choose of the scene if you select an area totally or partially contained in the path/row.

**Comment by gabriel-russo** [comment]
Grades disponíveis neste link do INPE: http://www.obt.inpe.br/OBT/assuntos/catalogo-cbers-amz-1
